terraform/workspace_variables/qa.tfvars

```diff
@@ -0,0 +1,13 @@
+# PaaS
+paas_sso_code                      = ""
+paas_app_environment               = "qa"
+paas_rails_env                     = "qa"
+paas_space_name                    = "bat-qa"
+paas_app_start_timeout             = "180"
```

Comment:
This should be the same for all environments. We could also hardcode it in the module.